packages/zoe/src/contractSupport/priceAuthorityQuoteMint.js

```diff
@@ -0,0 +1,52 @@
+import { Fail } from '@agoric/assert';
+import {
+  AssetKind,
+  makeDurableIssuerKit,
+  prepareIssuerKit,
+} from '@agoric/ertp';
+import { makeScalarBigMapStore } from '@agoric/vat-data';
+
+/**
+ *
+ * @param {import('@agoric/vat-data').Baggage} baggage
+ * @param {ERef<Mint<'set'>>} [providedQuoteMint]
+ */
+export const prepareQuoteMint = (baggage, providedQuoteMint) => {
```

Comment:
when we drop the privateArg (see below) I think this will be better factored as a `provideQuoteMint`.